edx_exams/apps/core/rest_utils.py

```diff
@@ -0,0 +1,37 @@
+"""
+Utils for reading and writing data to other services using REST.
+"""
+from django.conf import settings
+from edx_rest_api_client import client as rest_client
+
+
+# pylint: disable=inconsistent-return-statements
+def make_request(method, url, client, **kwargs):
```

Comment:
Question: is there a reason that a method needs to be created for this? As in, is there not an existing method that does this? Is there something custom happening here? Thanks for clarifying!

Reply:
not too much going on here really it's just to standardize calling `OAuthAPIClient.request()` and keeping that code alongside the `OAuthAPIClient` initialization. Also this throws an exception for non 200 responses so implementers don't have to check for that.